test/smart-wallet/onboardings-test.ts

```diff
@@ -372,15 +370,7 @@ describe("Wallet deployment cost estimation in various onbaording flows", functi
 
     const encodedEstimate = gasEstimatorCustomInterface.encodeFunctionData(
       "estimate",
-      [
-        expected,
-        walletFactory.address,
-        owner,
-        entryPoint.address,
-        handler.address,
-        3,
-        txnData,
-      ]
+      [expected, walletFactory.address, owner, 3, txnData]
```

Comment:
Why `_index` is `3` here?

Reply:
sometime we need a wallet that's not already deployed as part of current test block execution.

Reply:
because one with index 0 has been deployed already. and i am preforming another test